src/tasks/WorkloadBuildTasks/PackageInstaller.cs

```diff
@@ -123,7 +123,7 @@ private static string GenerateProject(IEnumerable<PackageReference> references)
             projectFileBuilder.Append(@"
 <Project Sdk=""Microsoft.NET.Sdk"">
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
can we use NETCoreAppMaximumVersion to avoid hardcoding a version?

```suggestion
        <TargetFramework>net$(NETCoreAppMaximumVersion)</TargetFramework>
```

Reply:
Do IIUC that when running with a 9.0 sdk this would be `net9.0`, and with 8.0 sdk it will be `net8.0`? Running this on an unrelated build I do see `net8.0` using a 8.0 sdk.